app/helpers/means_report_helper.rb

```diff
@@ -0,0 +1,50 @@
+module MeansReportHelper
```

Comment:
It feels like there is a Presenter/ViewComponent sort of object in here, but a helper is probably fine for now!